build-logic/src/main/kotlin/LicenseFileValidation.kt

```diff
@@ -58,7 +58,7 @@ class LicenseFileValidation : DependencyFilter {
   override fun filter(data: ProjectData?): ProjectData {
     data!!
 
-    val rootLicenseFile = data.project.rootProject.file("LICENSE").readText()
+    val rootLicenseFile = data.project.rootProject.file("LICENSE.bin").readText()
```

Comment:
`.bin` implies it's a binary file - maybe `DEPENDENCY-LICENSES.txt` or just `DEPENDENCY-LICENSES`?

Reply:
That's correct: `LICENSE.bin` is targeted to be included in binary distributions (like `polaris-service` tar/zip).
No problem to rename as `LICENSE-BINARY-DEPENDENCY` for instance.

Reply:
Then let's rename `LICENSE.bin` to `DEPENDENCY-LICENSE`, because `.bin` usually means "this" is a binary file.

Reply:
This PR creates a new license file. Do we need two license files. I think the one including the dependencies license is good enough.

Reply:
Yes we need two because we have a source distribution and a binary distribution.

If we don't distribute the binary distribution then only LICENSE would be enough. But as we have the polaris-service tar, to distribute it, we need the license for binaries included in this distribution.

Reply:
For binary distribution, do we need to aggregate all third-party licenses into a text file, as the third party jar files have them already?